weave/conftest.py

```diff
@@ -345,8 +345,9 @@ def int_to_ext(self, key, default):
         return self._int_to_ext_map[key]
 
 
-def simple_hash(s: str) -> str:
-    return hashlib.sha256(s.encode()).hexdigest()
+def b64(s: str) -> str:
```

Comment:
This change only changes the test harness id mapping. Now we have more formal requirements on the shape of our ids this needed to be updated